app/components/NavigationCard.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from "next/link";
+import Card from "./Card";
+import cn from "@/lib/helpers/cn";
+
+/**
+ * A Card containing a main navigation route in the main page.
+ * 
+ * @param title The Card title.
+ * @param body The Card's body text.
+ * @param href Where the Card navigates to.
+ * @param adornment A Small box indicating an ongoing event. Append like a property. 
+ */
+export default function NavigationCard({
+  adornment,
+  title,
+  body,
+  href,
+}: {
+  adornment?: React.ReactNode;
+  title: string;
+  body: string;
+  href: string;
+}) {
+  return (
+    <Link href={href} className="iex-navigation-card">
+      <Card className="border-cream relative aspect-2/1 flex-col items-center border-1 z-20">
+        <span
+          className={cn(
+            "text-yellow text-center text-headline-lg",
+            "sm:text-display-sm font-bold font-stretch-50% italic",
+            "[text-shadow:_4px_4px_0px_var(--color-bright-red)]",
+          )}
```

Comment:
- This should actually be `display-md`. Check the Dev Mode sidebar and it should say in the code section what font size / leading an element is.
- Why is there a breakpoint here? We’re literally only designing for mobile.
- `font-stretch` is deprecated. You’re supposed to set the [`"wdth"` font variation setting](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variation-settings#registered_and_custom_axes). Here’s [for Tailwind](https://tailwindcss.com/docs/font-family#customizing-your-theme).
  - You forgot to add `axes: ["wdth"]` to the NextFont options in `app/layout.tsx` which is probably why the utility you added had no effect.